website/source/docs/connect/proxies/built-in.md

```diff
@@ -15,7 +15,7 @@ Below is a complete example of all the configuration options available
 for the built-in proxy. Note that only the `service.connect.proxy.config` and
 `service.connect.proxy.upsteams[].config` maps are being described here, the
 rest of the service definition is shown for context but is [described
-elsewhere](/docs/connect/proxies.html#managed-proxies).
```

Comment:
@hashicorp/consul-docs Is the example below actually using a managed proxy and should it be updated if so?

Reply:
Nope this is showing the raw configuration of a proxy-kind registration with all valid options. To make that clearer we could add `"kind": "connect-proxy" to the top level.